### PR TITLE
bust local team cache on incoming gregor

### PR DESCRIPTION
**Pull request by maxtaco**
- leave TODO: which is to implement ~~ForceRepollRecursive~~ `notifyTeamRename`
- make a test to test it

### CODE REVIEW

**Comment by maxtaco** [comment]
@mlsteele what's the best way to book-keep this todo? Maybe in whatever JIRA ticket is going to handle it?

File: go/teams/get.go [hunk 1]
```diff
@@ -81,3 +89,12 @@ func GetForApplicationByName(ctx context.Context, g *libkb.GlobalContext, name k
 func GetForChatByStringName(ctx context.Context, g *libkb.GlobalContext, s string, refreshers keybase1.TeamRefreshers) (*Team, error) {
 	return GetForApplicationByStringName(ctx, g, s, keybase1.TeamApplication_CHAT, refreshers)
 }
+
+func ForceTeamRefresh(ctx context.Context, g *libkb.GlobalContext, id keybase1.TeamID, changes keybase1.TeamChangeSet) error {
+	_, err := Load(ctx, g, keybase1.LoadTeamArg{
+		ID:                   id,
+		ForceRepoll:          true,
+		ForceRepollRecursive: changes.Renamed,
```
Comment (mlsteele) [inline go/teams/get.go]: We could only do `ForceRepollRecursive` when `changes.Renamed`, right? Doesn't seem like a big deal though.

**Comment by maxtaco** [comment]
Actually,  hmrg, the message was never getting dismissed!  But I think the plan was not to dismiss these guys but rather to let them expire.

**Comment by maxtaco** [comment]
ok @mlsteele please look again, this has the changes we talked about.

**Pull request merged**